go.mod

```diff
@@ -49,3 +51,5 @@ replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.0+inc
 // 78d2af792bab is the commit tagged as v12.0.0.
 
 replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
+
+replace github.com/tonobo/mtr v0.1.1-0.20210410081413-c8f9fe0b48b9 => ../mtr
```

Comment:
I'll need to point this back to the actual library. I had a local copy running to experiment with changing the mtr library